src/Modules.js

```diff
@@ -2,26 +2,26 @@
 * Core modules and their info files
 */
 module.exports = {
+  'average': require('./modules/Average'),
+  'blend': require('./modules/Blend'),
+  'blur': require('./modules/Blur'),
+  'brightness': require('./modules/Brightness'),'channel': require('./modules/Channel'),
```

Comment:
Can you explain why you wrote this ```'channel': require('./modules/Channel'), ``` because it is already present in next line? 
So please remove this duplicate value